web/src/portico/tabbed-instructions.ts

```diff
@@ -58,9 +58,9 @@ export function activate_correct_tab($codeSection: JQuery<HTMLElement>): void {
     const $active_list_items = $li.filter(".active");
     if (!$active_list_items.length) {
         $li.first().addClass("active");
-        const language = $li.first()[0].dataset.language;
-        if (language) {
-            $blocks.filter("[data-language=" + language + "]").addClass("active");
+        const tab_key = $li.first()[0].dataset.tab_key;
+        if (tab_key) {
+            $blocks.filter("[data-tab_key=" + tab_key + "]").addClass("active");
```

Comment:
This should match the `data-tab-key` HTML data attribute from the templates in `tabbed_sections.py`.

```suggestion
            $blocks.filter("[data-tab-key=" + tab_key + "]").addClass("active");
```